packages/graphql/tests/integration/directives/fulltext.int.test.ts

```diff
@@ -74,672 +94,2415 @@ describe("@fulltext directive", () => {
+                query {
+                    ${queryType}(phrase: "a name", limit: 0.5) {
+                        score
+                        ${personTypeLowerFirst} {
+                            name
+                            born
+                        } 
+                    }
+                }
+            `;
+            const gqlResult = await graphql({
+                schema: generatedSchema,
+                source: query,
+                contextValue: {
+                    driver,
+                    driverConfig: { database: databaseName },
+                },
+            });
+
+            expect((gqlResult.errors as any[])[0].message).toBe(`Int cannot represent non-integer value: ${notAnInt}`);
+        });
+        test("Throws error if if invalid argument is suplied", async () => {
+            const invalidArgument = "not_a_valid_argument";
+            const query = `
+                query {
+                    ${queryType}(phrase: "a name", ${invalidArgument}: [1, 2, 3]) {
+                        score
+                        ${personTypeLowerFirst} {
+                            name
+                            born
+                        } 
+                    }
+                }
+            `;
+            const gqlResult = await graphql({
+                schema: generatedSchema,
+                source: query,
+                contextValue: {
+                    driver,
+                    driverConfig: { database: databaseName },
+                },
+            });
+
+            expect((gqlResult.errors as any[])[0].message).toStartWith(
+                `Unknown argument "${invalidArgument}" on field`
+            );
+        });
+        test("Sorting by score when the score is not returned", async () => {
+            const query = `
+                query {
+                    ${queryType}(phrase: "a different name", sort: { score: ASC }) {
+                        ${personTypeLowerFirst} {
+                            name
+                        } 
+                    }
+                }
+            `;
+            const gqlResult = await graphql({
+                schema: generatedSchema,
+                source: query,
+                contextValue: {
+                    driver,
+                    driverConfig: { database: databaseName },
+                },
+            });
+
+            expect(gqlResult.errors).toBeFalsy();
+            expect(gqlResult.data?.[queryType]).toEqual([
+                {
+                    [personTypeLowerFirst]: {
+                        name: person3.name,
+                    },
+                },
+                {
+                    [personTypeLowerFirst]: {
+                        name: person1.name,
+                    },
+                },
+                {
+                    [personTypeLowerFirst]: {
+                        name: person2.name,
+                    },
+                },
+            ]);
```

Comment:
Just to confirm, this is the correct order? The score sort is applied within the Cypher in this scenario?

Reply:
Yes, this is the correct order